numpy/random/_generator.pyx

```diff
@@ -2004,7 +2004,7 @@ cdef class Generator:
         -----
         The probability density for the Pareto distribution is
 
-        .. math:: p(x) = \\frac{am^a}{x^{a+1}}
+        .. math:: p(x) = \\frac{am^a}{{x+1}^{a+1}} , x \ge m
```

Comment:
```suggestion
        .. math:: p(x) = \\frac{a}{{x+1}^{a+1}} , x \ge 0
```
As mentioned in the issue thread, there's no `m` in the true PDF, so it should be removed.